BigQuery/src/BigQueryClient.php

```diff
@@ -683,6 +683,30 @@ public function timestamp(\DateTimeInterface $value)
         return new Timestamp($value);
     }
 
+    /**
+     * Create a Numeric object.
+     *
+     * Numeric represents a value with a data type of
+     * [Numeric](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#numeric-type).
+     *
+     * It supports a fixed 38 decimal digits of precision and 9 decimal digits of scale, and values
+     * are in the range of -99999999999999999999999999999.999999999 to
+     * 99999999999999999999999999999.999999999.
+     *
+     * Example:
+     * ```
+     * $numeric = $bigQuery->numeric('99999999999999999999999999999999999999.999999999');
+     * ```
+     *
+     * @param string $value The Numeric value.
```

Comment:
This is documented as `string|int|float` on `Numeric::__constructor()` now, so it should be updated here as well.

Reply:
Done